tests/testthat/test-html-tweak.R

```diff
@@ -354,3 +354,69 @@ test_that("activate_navbar()", {
       xml2::xml_find_first(navbar, ".//li[contains(@class, 'active')]")
   )
 })
+
+# tabsets -------------------------------------------------------------
+
+test_that("tweak_tabsets() default", {
+  html <- '<div id="results-in-tabset" class="section level2 tabset">
+<h2 class="hasAnchor">
+<a href="#results-in-tabset" class="anchor" aria-hidden="true"></a>Results in tabset</h2>
+<div id="tab-1" class="section level3">
+<h3 class="hasAnchor">
+<a href="#tab-1" class="anchor" aria-hidden="true"></a>Tab 1</h3>
+<p>blablablabla</p>
+<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
```

Comment:
Is it important to have source code in here? Otherwise it would be better to keep the test as short and simple as possible.

Reply:
I can't find of another solution right now https://github.com/r-lib/pkgdown/issues/1725#issuecomment-864003811